.github/workflows/static-code-analysis.yml

```diff
@@ -4,12 +4,15 @@ on:
     branches: [master]
   pull_request:
     branches: [master]
+  pull_request_target:
+    [opened, synchronize, reopened]
```

Comment:
This is default, it can be omitted.
Is there a particular reason why pr_target should be run against all branches where push/pr against master only?

Reply:
Changed to check only master branch.